File: C++/salary.cpp
#include <iostream>
#include <cstdio>
using namespace std;
int main(){
    int employee, hours;
    double money, salary;
    cin >> employee >> hours >> money;
    salary = money * hours;
    cout << "NUMBER =  " << employee << endl;
    printf("SALARY = U$ %.2lf\n", salary);
    return 0;
}

File: Java/Salary.java
import java.util.Scanner;
public class Salary {
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int number, hours;
        double money;
        number = scan.nextInt();
        hours = scan.nextInt();
        money = scan.nextDouble();
        System.out.println("NUMBER = " + number);
        System.out.format("SALARY = U$ %.2f\n", (hours*money));
    }
}
